### PR TITLE
Dev kmr

### PR DESCRIPTION
**Pull request by k-rister**

### CODE REVIEW

File: .github/actions/integration-tests/mv-params/oslat-2.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "global-options": [
+	{
+	    "name": "required",
+	    "params": [
+                { "arg": "duration", "vals": [ "10", "15" ], "role": "client" },
+	        { "arg": "rtprio", "vals": [ "2" ], "role": "client" }
```
Comment (rafaelfolco) [inline .github/actions/integration-tests/mv-params/oslat-2.json:7]: is it a testcase rtprio=2 ?

**Review by rafaelfolco — APPROVED** [review]

File: .github/actions/integration-tests/mv-params/oslat-2.json (hunk 1 above)
Comment (k-rister) [reply .github/actions/integration-tests/mv-params/oslat-2.json:7]: It is debatable whether or not we would want to actually run this "for real", I was just trying to exercise the different options with varying parameters for some differences between the iterations.

Note, I wouldn't expect any of the results from these tests to be good.

File: .github/actions/integration-tests/run-ci-stage2 [hunk 2]
```diff
@@ -46,11 +64,62 @@ case "${CI_RUN_ENVIRONMENT}" in
...
+    cmd="podman rmi --all"
+    echo "${cmd}"
+    ${cmd}
+    stop_github_group
+
+    header="Removing Crucible installed/created files"
+    start_github_group "${header}"
+    echo -e "### ${header} ###\n"
+    echo
+    cmd="rm -Rfv /opt/crucible* /var/lib/crucible* /etc/sysconfig/crucible  /root/.crucible /etc/profile.d/crucible_completions.sh /home/crucible-containers"
```
[the remark this answers is not in the document]
Comment (k-rister) [reply .github/actions/integration-tests/run-ci-stage2:104]: I'm not sure I understand what you are asking here.

[the remark this answers is not in the document]
Comment (rafaelfolco) [reply .github/actions/integration-tests/run-ci-stage2:104]: > I'm not sure I understand what you are asking here.

you cleaning up and you are removing crucible, I assume somewhere else you re-use the env / job but reinstall crucible for the the next test ?

[the remark this answers is not in the document]
Comment (k-rister) [reply .github/actions/integration-tests/run-ci-stage2:104]: Ahh, ok, I understand the question now.  When using Github hosted runners, which is everything we have done up until now, the environment is ephemeral and thrown away after it runs a job.  This means there is no persistence of the environment from one job to the next so the CI code expects a clean environment each time.  The same does not apply to self hosted runners, the environment does persist from one job to the next so we need to clean up at the end of a run to ensure that a job doesn't leave pieces behind that will interfere with the next job.

**Pull request merged**